# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: dom/media/mediasink/MediaSink.h
# Commit: 3b70213ee976
# Full Hash: 3b70213ee976d2b7d3b0f17876b833149e085ffd
# Author: Paul Adenot <paul@paul.cx>
# Date: 2022-05-24 21:44:48
# Regressor Bug: 1743834
# File Overlap Count: 1
# Description:
#   Bug 1743834 - Make GetClock() non-const on MediaSink, and allow the AudioSink to modify the AudioQueue. r=alwu,media-playback-reviewers
#   
#   Differential Revision: https://phabricator.services.mozilla.com/D133642
# ==============================================================================

diff -r 6323b526b0ac -r 3b70213ee976 dom/media/mediasink/MediaSink.h
--- a/dom/media/mediasink/MediaSink.h	Tue May 24 12:59:10 2022 +0000
+++ b/dom/media/mediasink/MediaSink.h	Tue May 24 13:09:04 2022 +0000
@@ -60,8 +60,7 @@
   // aTimeStamp returns the timeStamp corresponding to the returned position
   // which is used by the compositor to derive the render time of video frames.
   // Must be called after playback starts.
-  virtual media::TimeUnit GetPosition(
-      TimeStamp* aTimeStamp = nullptr) const = 0;
+  virtual media::TimeUnit GetPosition(TimeStamp* aTimeStamp = nullptr) = 0;
 
   // Return true if there are data consumed but not played yet.
   // Can be called in any state.